functional_tests/features/bundle_ui.feature

```diff
@@ -0,0 +1,211 @@
+
+
+#---- Bundle Cannot preview UI Tests -----
+Scenario Outline: A User cannot preview a bundle due to not member of  associated preview-team
+    Given there is a <role> user
+    And there is a <creator_role> user
+    And there is a Statistical Analysis page approved by <creator_role>
+    And there is a Release Calendar page approved by <creator_role>
+    And there is a preview team
+    And there are <number_of_bundles> bundles with <bundle_details>
+    When the <role> logs in
+    Then the logged in user cannot preview a bundle
+
+ Examples: bundles
+     | number_of_bundles | role                       | creator_role        | bundle_details                                                                                                                                       |
+     | 1                 | Viewer                     | Publishing Admin    | {"role": "Viewer",             "creator_role": "Publishing Admin", "status": "In_Review", "preview_teams": false,  "add_rel_cal": true, "add_stat_page": true}|
+
+
+#----- Bundle Approve UI Tests -----
+
+Scenario Outline: A user can approve a bundle
+    Given there is a <role> user
+    And there is a <creator_role> user
+    And there is a Statistical Analysis page approved by <creator_role>
+    And there is a Release Calendar page approved by <creator_role>
+    And there is a preview team
+    And the <role> is a member of the preview team
+    And there are <number_of_bundles> bundles with <bundle_details>
+    When the <role> logs in
+    Then the <role> can preview the Release Calendar page
+    And the <role> can preview the Statistical Analysis page
+    And the <role> can preview bundle
+    And the logged in user can approve a bundle
+
+    Examples: bundles
+        | number_of_bundles | role                       | creator_role        | bundle_details                                                                                                                                                 |
+#        | 1                 | Publishing Admin           | Publishing Officer  | {"role": "Publishing Admin",   "creator_role": "Publishing Officer", "status": "In_Review", "preview_teams": true, "add_rel_cal": true, "add_stat_page": true} |
+#        | 1                 | Publishing Admin           | Publishing Admin    | {"role": "Publishing Admin",   "creator_role": "Publishing Admin",   "status": "In_Review", "preview_teams": true, "add_rel_cal": true, "add_stat_page": true} |
+#        | 1                 | Publishing Officer         | Publishing Officer  | {"role": "Publishing Officer", "creator_role": "Publishing Officer", "status": "In_Review", "preview_teams": true, "add_rel_cal": true, "add_stat_page": true} |
+#        | 1                 | Publishing Officer         | Publishing Admin    | {"role": "Publishing Officer", "creator_role": "Publishing Admin",   "status": "In_Review", "preview_teams": true, "add_rel_cal": true, "add_stat_page": true} |
```

Comment:
If there is a reason why you have this commented out, it would be prudent to either leave a comment explaining why, or update the PR description so that this is clear.

Same on line 62, 63 and 211.